<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Acerca</class>
 <widget class="QDialog" name="Acerca">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>653</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Acerca de Bio-Feedback</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelIMAGE">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/images/home.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelVersion">
         <property name="text">
          <string>Version:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelQtVersion">
         <property name="text">
          <string>QT Version</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelAutor">
         <property name="text">
          <string>Creado por:
Héctor Gabriel Peredo Urbina
Ingeniero Civil Informático
Universidad Católica del Maule</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBuild">
         <property name="text">
          <string>Compile Date</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
